codegen/gateway.go

```diff
@@ -125,7 +119,7 @@ func NewClientSpec(
 	if err := json.Unmarshal(instance.JSONFileRaw, &clientConfig); err != nil {
 		return nil, errors.Wrapf(
 			err,
-			"Could not parse class config json file %s: ",
+			"Could not parse class config json file: %s",
```

Comment:
I prefer `%q` most of the time to make the interesting part stand out, but here the json file path will be long, and wrapping it in quotes doesn't serve that purpose well.